src/net.cpp

```diff
@@ -575,6 +578,9 @@ bool CNode::Unban(const CSubNet &subNet) {
     if (setBanned.erase(subNet))
     {
         setBannedIsDirty = true;
+
+        DumpBanlist(); //store banlist to disk
```

Comment:
After looking through the code more, I think writing to disk for unbanning a node is unnecessary.  The only time we ever read this file from disk is when we start the node, which immediately calls `SweepBanned` to clear out expired bans.